src/lwc/geFormField/geFormField.js

```diff
@@ -662,8 +662,7 @@ export default class GeFormField extends LightningElement {
 
         return siblingRecordTypeId ||
             this.parentRecordRecordTypeId() ||
-            this.defaultRecordTypeId() ||
-            null;
+            this.defaultRecordTypeId();
```

Comment:
@psadie-sfdo does this ever return undefined?  I think the || null was there because returning null explicitly would be better than undefined...

Reply:
Yeaah, it is possible to return undefined here if formState is set before the wired getObjectInfo returns with data. The specific bug we're trying to cover with this PR is due to `geFormField` being used in the allocation row component. Those instances of `geFormField` don't receive the form state. Currently of the opinion that that's a bigger issue we need to tackle in a separate work item.

Went ahead and pushed a commit to explicitly return null here again if no record type id is found